src/stores/Frame/FrameStore.ts

```diff
@@ -193,7 +193,11 @@ export class FrameStore {
     }
 
     @computed get maxMip(): number {
-        return Math.pow(2, Math.ceil(Math.log2(this.frameInfo.fileInfoExtended.width / TILE_SIZE)));
+        if (this.frameInfo.fileInfoExtended.width < 256 || this.frameInfo.fileInfoExtended.height < 256) {
```

Comment:
use `TILE_SIZE` instead of 256. We don't use height in the calculation, so maybe we don't need to check `height < 256`.

Reply:
The suggestion is practiced with the modification of [431aa93](https://github.com/CARTAvis/carta-frontend/pull/1980/commits/431aa938b8cc56909ea1f9f023bdd746d4eb56cf).